ee/billing/billing_manager.py

```diff
@@ -105,6 +160,11 @@ def get_billing(self, organization: Optional[Organization], plan_keys: Optional[
             product["current_usage"] = current_usage
             product["percentage_usage"] = current_usage / usage_limit if usage_limit else 0
 
+            # Also update the tiers
+            if product["tiers"]:
+                product["tiers"] = compute_usage_per_tier(current_usage, product["projected_usage"], product["tiers"])
+                product["current_amount_usd"] = sum_total_across_tiers(product["tiers"])
```

Comment:
I think this needs to happen for addons as well, and the updating needs to depend on the addon usage_key since now addons can have a different usage key than their parent.